engine/classes/ElggGroup.php

```diff
@@ -523,6 +523,12 @@ protected function create() {
 		global $CONFIG;
 		
 		$guid = parent::create();
+		if (!$guid) {
+			// @todo this probably means permission to create entity was denied
+			// Is returning false the correct thing to do
```

Comment:
I'm just going to answer the question "yes", now we don't need the comment in either place :)